crates/turborepo-lib/src/task_graph/visitor.rs

```diff
@@ -343,7 +343,7 @@ impl<'a> Visitor<'a> {
                 match exit_status {
                     // The task was successful, nothing special needs to happen.
                     ChildExit::Finished(Some(0)) => {
-                        let _summary = tracker.build_succeeded(0);
+                        tracker.build_succeeded(0).await;
```

Comment:
Very silly, but very important (for correctness) change.

Reply:
Comment here as to why?

Reply:
Wasn't awaiting a future so it never got ran 🙃

Reply:
Haha well that explains it. I was wondering why we didn't await but if its just because of 'integration' then no comment needed.